packages/amplication-client/src/Project/ArchitectureConsole/ModelOrganizer.tsx

```diff
@@ -59,7 +59,13 @@ const edgeTypes = {
 };
 
 export default function ModelOrganizer() {
-  const { currentProject } = useAppContext();
+  const { currentProject, resetPendingChangesIndicator } = useAppContext();
+
+  useEffect(() => {
+    if (!resetPendingChangesIndicator) return;
+    resetChanges();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Is this on purpose?
Shall we add resetChanges to the deps?

Reply:
@overbit  Yes, It is; it causes an infinity loop. this is just a callback function that reset all the changes.